localstack-core/localstack/services/dynamodb/resource_providers/aws_dynamodb_globaltable.py

```diff
@@ -239,6 +239,12 @@ def create(
                 # rename bool attribute to fit boto call
                 sse_specification["Enabled"] = sse_specification.pop("SSEEnabled")
 
+            if stream_spec := model.get("StreamSpecification"):
+                create_params["StreamSpecification"] = {
+                    "StreamEnabled": True,
+                    **(stream_spec or {}),
```

Comment:
Nit: which values for `stream_spec` will cause the `or` case to be invoked?

Reply:
Nice nit. It shouldn't be possible to hit the `or` case (given the assignment expression). `{}` also won't be a good default value anyway.